kubectl-fdb/cmd/exclude.go

```diff
@@ -0,0 +1,118 @@
+/*
+ * exclude.go
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2020 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"log"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	ctx "context"
+	fdbtypes "github.com/FoundationDB/fdb-kubernetes-operator/api/v1beta1"
+)
+
+// excludeCmd represents the exclude command of the fdb cli
+var excludeCmd = &cobra.Command{
```

Comment:
In an additional PR we could also add the following features:

- exclude all instances of a cluster
- exclude all instances with a specific label (e.g. all storage instances)

I will open an follow up issue for this task.

Reply:
Wouldn't those operations fail due to excluding too much of the cluster?

Reply:
Ah, I was confused about what the `exclude` operation does. I think we need to be consistent in our use of `exclude` and `replace`. We may want to talk about this more out-of-band to get the right terminology.

Reply:
I adjusted the name to `remove` like discussed "offline"